exonum-java-binding/core/src/main/java/com/exonum/binding/util/LibraryLoader.java

```diff
@@ -20,29 +20,81 @@
 import org.apache.logging.log4j.Logger;
 
 /**
- * A loader of the native shared library with Exonum framework bindings.
+ * A loader of the native shared library with Exonum framework bindings. It loads the native
+ * library and also verifies that it is compatible with the Java classes. The native library
+ * is compatible iff it has exactly the same version as this Java library. The revision
```

Comment:
Typo: iff -> if

Reply:
It isn't: "if and only if"

Reply:
https://en.oxforddictionaries.com/definition/iff